package-lock.json

```diff
@@ -2758,7 +2758,7 @@
         },
         "lodash": {
           "version": "1.0.2",
-          "resolved": "https://registry.npmjs.org/lodash/-/lodash-1.0.2.tgz",
+          "resolved": "http://registry.npmjs.org/lodash/-/lodash-1.0.2.tgz",
```

Comment:
🤔 Do we know why the scheme changed from `https` to `http`?

Reply:
Thanks David, I did not spot this. I realised, from talking to Nick, that I had a different version of Node installed on my computer, so therefore it updated package-lock.json file with these new dependencies . I have done a git-reset and removed these changes and now installed the correct Node on my computer. I shall make sure I check my commits more thoroughly next time before I push! :)

Reply:
@sldblog Hi David, I am just replying to your comment below about the readme using Node 8. I had to quiz Nick about that as I wasn't sure. Apparently when you install Node 8, it installs the latest minor version of node 8. So that will be the same version that package-lock.json needs.